save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/vulnerabilities/LnkVulnerabilityUser.kt

```diff
@@ -0,0 +1,20 @@
+package com.saveourtool.save.entities.vulnerabilities
+
+import com.saveourtool.save.entities.User
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.*
+
+/**
+ * @property vulnerabilityId
+ * @property user in vulnerability with [vulnerabilityId]
+ */
+@Entity
+class LnkVulnerabilityUser(
+
+    var vulnerabilityId: Long,
```

Comment:
why it was refactored to a single id?

Reply:
Both the Vulnerability and the User entities contain fields with the ManyToOne type. The entity LinkVulnerabilityUser will then contain two such fields, but in Hibernate this is prohibited and throw an error.

Reply:
Yes
_Fetching two or more Bags at the same time on an Entity could form a Cartesian Product. Since a Bag doesn't have an order, Hibernate would not be able to map the right columns to the right entities. Hence, in this case, it throws a MultipleBagFetchException._

https://www.baeldung.com/java-hibernate-multiplebagfetchexception#cause-of-multiplebagfetchexception

Reply:
I meant in code :)

Reply:
Ok, adding now in the next pr )